### ISSUE TITLE
enhancement:  query data across account hierarchy

### ISSUE THREAD

**Issue by chrislittle** [CenturyLinkCloud/clc-go-cli#91]
It would be great if we could implement a 'crawl' of an account hierarchy for CLI actions.  

for example, I want a list of all servers across my parent and all subs.  right now you must always specify --account-alias or it just lists the parent account data.

**Comment by richardcase** [comment]
Great idea. Do you have any preferences on how we specify this on the command line. For example:

`clc server list --data-center VA1 --account-alias CSEM --include-subaccounts `

**Comment by chrislittle** [comment]
well we already have --account-alias on some commands (not sure how many so if you can check).  perhaps we just use --account-alias and it can be a specific alias or 'all'  ... @gavin-lai your thoughts?  i'm thinking if we can avoid another function lets do it.

**Comment by gavin-lai** [comment]
Agree, the all option sound go to get all sub accounts information.
